react-native-cli/index.js

```diff
@@ -145,9 +156,21 @@ function createProject(name) {
       process.exit(1);
     }
 
-    var args = [projectName].concat(process.argv.slice(4));
+    var cli = require(CLI_MODULE_PATH());
+    cli.init(root, projectName);
+  });
+}
+
+function runVerbose(root, projectName) {
+  var proc = spawn('npm', ['install', '--verbose', '--save', 'react-native'], {stdio: 'inherit'});
+  proc.on('close', function (code) {
+    if (code !== 0) {
+      console.error('`npm install --save react-native` failed');
+      return;
+    }
```

Comment:
Is this needed? npm will probably tell you it failed. Then you don't need spawn.